tests/src/test/scala/limits/ThrottleTests.scala

```diff
@@ -298,10 +298,25 @@ class NamespaceSpecificThrottleTests
     with BeforeAndAfterAll
     with LocalHelper {
 
+  val defaultAction = Some(TestUtils.getTestActionFilename("hello.js"))
+
   val wskadmin = new RunWskAdminCmd {}
   val wsk = new WskRest
 
-  val defaultAction = Some(TestUtils.getTestActionFilename("hello.js"))
+  def sanitizeNamespaces(namespaces: Seq[String], expectedExitCode: Int = SUCCESS_EXIT) = {
+    namespaces.foreach { ns =>
+      try {
+        disposeAdditionalTestSubject(ns, expectedExitCode)
+        withClue(s"failed to delete temporary limits for $ns") {
+          wskadmin.cli(Seq("limits", "delete", ns), expectedExitCode)
```

Comment:
`expectedExitCode` will make no difference if you try/catch and ignore everything, or did I miss something?

Reply:
The cli cmds shouldn’t fail with expected exit set to don’t care but could fail if set to success. The try slows other ids to be cleaned. 

Are you thinking to force failure vs logging only?

Reply:
Yep, this was towards "force failure vs. logging only"

Reply:
i'll capture the exceptions and throw one at the end - at least to allow other ids to be cleaned first failing fast.